tools/autograd/gen_python_functions.py

```diff
@@ -233,25 +247,6 @@ def create_python_bindings(python_functions, has_self, is_module=False):
         'ScalarType': 'scalartypeWithDefault',
     }
 
-    def first_tensor_arg(arguments):
-        for arg in arguments:
-            if arg['simple_type'] in {'Tensor', 'TensorList'}:
-                return arg['name']
-        return None
-
-    def auto_gpu(option, has_device_bind):
```

Comment:
This has moved to `aten/src/ATen/function_wrapper.py` since `AutoGPU` is now in ATen